docs/okidoc-site-md-components.md

```diff
@@ -60,6 +60,6 @@ Then use components in any `md` file:
 ```
 
 > NOTE:
-> md components logic is inspired by [rehype-react](https://using-remark.gatsbyjs.org/custom-components/),
+> Logic for md components is inspired by [rehype-react](https://using-remark.gatsbyjs.org/custom-components/),
 > and based on [hast-to-hyperscript](https://github.com/syntax-tree/hast-to-hyperscript),
 > so has the same caveats.
```

Comment:
Just a note IMO - since users might not know what those caveats are, a link here would be helpful... @bodia-uz

Reply:
You are right. Event with the link it's not clear enough, so we need to update this text with more details (in another PR). 
```suggestion
> so has the same [caveats](https://github.com/syntax-tree/hast-to-hyperscript#caveats).
```